src/test/java/seedu/address/logic/commands/SessionCommandTest.java

```diff
@@ -113,6 +113,31 @@ public void execute_invalidPersonIndexFilteredList_failure() {
         assertCommandFailure(sessionCommand, model, Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
 
+    @Test
+    public void execute_addSessionIgnoreCase_success() {
+        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+        final SessionCommand standardCommand = new SessionCommand(INDEX_FIRST_PERSON,
+                new Session(SESSION_STUB));
+        final SessionCommand ignoreCaseCommand = new SessionCommand(INDEX_FIRST_PERSON,
+                new Session("mON 08:00"));
```

Comment:
Usually we write the descriptions as final Strings (like SESSION_STUB) so it would be good to do the same for mON 08:00

Reply:
okkiee thank you